### PR TITLE
Add a new tool to add has_str flag to bugs with a STR

### PR DESCRIPTION
**Pull request by calixteman**
First implementation of #400.

### CODE REVIEW

File: auto_nag/scripts/has_str_no_hasstr.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    def set_autofix(self, bugs, with_str):
+        for bugid, one_comment in with_str.items():
+            bug = bugs[bugid]
+            if one_comment:
+                # we restrict this autofix to bugs with only one comment and no history
+                # to avoid a maybe inappropriate comment in a chat.
+                nick = bug['nick']
+                self.autofix_hasstr[bugid] = {
+                    'cf_has_str': 'yes',
+                    'comment': {
```
Comment (marco-c) [inline auto_nag/scripts/has_str_no_hasstr.py]: We could restrict this autofixer to only add `cf_has_str`, and then let the other autofixer (the one which will be added with #397) add the comment.

**Comment by sylvestre** [comment]
Needs rebasing and I don't want to give it a try :)

**Comment by sylvestre** [comment]
@calixteman ping?

**Comment by philipp-sumo** [comment]
seeing the bot get into action a couple of times now after this change is live, where it wasn't really enhancing the bug with helpful information  - i've left a comment in #400 about it.

**Pull request merged**